csharp/WriterProperties.cs

```diff
@@ -21,14 +28,41 @@ public void Dispose()
             Handle.Dispose();
         }
 
+        /// <summary>
+        /// The entity that created the file.
+        /// </summary>
         public string CreatedBy => ExceptionInfo.ReturnString(Handle, WriterProperties_Created_By, WriterProperties_Created_By_Free);
+        /// <summary>
+        /// The size of the data pages in bytes.
+        /// </summary>
         public long DataPageSize => ExceptionInfo.Return<long>(Handle, WriterProperties_Data_Pagesize);
+        /// <summary>
+        /// The <see cref="ParquetSharp.Encoding"/> used for dictionary index encoding.
+        /// </summary>
         public Encoding DictionaryIndexEncoding => ExceptionInfo.Return<Encoding>(Handle, WriterProperties_Dictionary_Index_Encoding);
+        /// <summary>
+        /// The <see cref="ParquetSharp.Encoding"/> used for dictionary page encoding.
+        /// </summary>
         public Encoding DictionaryPageEncoding => ExceptionInfo.Return<Encoding>(Handle, WriterProperties_Dictionary_Page_Encoding);
+        /// <summary>
+        /// The maximum dictionary page size in bytes.
+        /// </summary>
         public long DictionaryPagesizeLimit => ExceptionInfo.Return<long>(Handle, WriterProperties_Dictionary_Pagesize_Limit);
+        /// <summary>
+        /// The <see cref="ParquetSharp.FileEncryptionProperties"/> used for writing encrypted files.
+        /// </summary>
         public FileEncryptionProperties FileEncryptionProperties => new FileEncryptionProperties(ExceptionInfo.Return<IntPtr>(Handle, WriterProperties_File_Encryption_Properties));
+        /// <summary>
+        /// The maximum number of rows in a row group.
+        /// </summary>
         public long MaxRowGroupLength => ExceptionInfo.Return<long>(Handle, WriterProperties_Max_Row_Group_Length);
+        /// <summary>
+        /// The version of the Parquet format to write.
+        /// </summary>
         public ParquetVersion Version => ExceptionInfo.Return<ParquetVersion>(Handle, WriterProperties_Version);
+        /// <summary>
+        /// The size of the write batch in bytes.
```

Comment:
```suggestion
        /// The number of records to batch together when writing.
```

Reply:
Whoops, I should've double-checked that one...